src/handlers/RF24DualLogHandler.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file RF24DualLogHandler.cpp
+ *
+ * Created on: 30 Mar 2021
+ *     Author: Witold Markowski (wmarkow)
+ *
+ * Copyright (C)
+ *    2021        Witold Markowski (wmarkow)
+ *
+ * This General Public License does not permit incorporating your program into
+ * proprietary programs.  If your program is a subroutine library, you may
+ * consider it more useful to permit linking proprietary applications with the
+ * library.  If this is what you want to do, use the GNU Lesser General
+ * Public License instead of this License.
+ */
+
+#include <handlers/RF24DualLogHandler.h>
+
+RF24DualLogHandler::RF24DualLogHandler(RF24LogHandler *handler1,
+      RF24LogHandler *handler2)
+{
+   this->handler1 = handler1;
+   this->handler2 = handler2;
+}
+
+void RF24DualLogHandler::log(uint8_t logLevel,
+      const __FlashStringHelper *vendorId, const char *message, va_list *args)
+{
+   // va_list can be iterated only once.
+   // Since we have here two handlers, we need to copy arguments.
+   va_list args2;
+   va_copy(args2, *args);
+
+   // redirect logs to wrapped handlers
+   handler1->log(logLevel, vendorId, message, args);
+   handler2->log(logLevel, vendorId, message, &args2);
+}
+
+void RF24DualLogHandler::log(uint8_t logLevel,
+      const __FlashStringHelper *vendorId, const __FlashStringHelper *message,
+      va_list *args)
+{
+   // va_list can be iterated only once.
+   // Since we have here two handlers, we need to copy arguments.
+   va_list args2;
+   va_copy(args2, *args);
+
+   // redirect logs to wrapped handlers
+   handler1->log(logLevel, vendorId, message, args);
+   handler2->log(logLevel, vendorId, message, &args2);
+}
+
+void RF24DualLogHandler::setLogLevel(uint8_t logLevel)
+{
+   handler1->setLogLevel(logLevel);
+   handler2->setLogLevel(logLevel);
+}
+
+void RF24DualLogHandler::write(uint8_t logLevel, const __FlashStringHelper *vendorId,
+         const char *message, va_list *args)
+{
+   // no implementation needed; log messages are redirected to handlers in custom log() method
```

Comment:
1. compiler complains that there are unused arguments. 
2. as stated in your comment, this method doesn't need to exist because its never called.

This relates to my objection about unnecessarily inheriting from `RF24LogHandler` (see my comment in _RF24DualLogHandler.h_ class declaration)

Reply:
This has been already reworked and those ```write()``` methods doesn't exist in ```RF24DualLogHandler```.